Tests/SwiftDocCTests/Indexing/NavigatorIndexTests.swift

```diff
@@ -402,7 +402,7 @@ Root
         let converter = DocumentationContextConverter(bundle: bundle, context: context, renderContext: renderContext)
         var results = Set<String>()
         
-        // Create an index 10 times to ensure we have not undeterministic behavior across builds
+        // Create an index 10 times to ensure we have not indeterministic behavior across builds
```

Comment:
```suggestion
        // Create an index 10 times to ensure we have not non-deterministic behavior across builds
```

(I think)

Reply:
Yes, that's correct. Indeterministic is also a word but it means something different.